clang/test/CodeGenObjC/constant-non-fragile-ivar-offset.m

```diff
@@ -17,9 +23,71 @@ @implementation StaticLayout {
 -(void)meth {
   static_layout_ivar = 0;
   // CHECK-NOT: load i64, ptr @"OBJC_IVAR_$_StaticLayout
+  // CHECK: getelementptr inbounds i8, ptr %0, i64 20
 }
 @end
 
+// Ivars declared in the @interface
+@interface SuperClass : NSObject
+@property (nonatomic, assign) int superClassProperty;
+@end
+
+@implementation SuperClass {
+  int superClassIvar; // Declare an ivar
+}
+- (void)superClassMethod {
+    _superClassProperty = 42;
+    superClassIvar = 10;
+    // CHECK-NOT: load i64, ptr @"OBJC_IVAR_$_SuperClass
```

Comment:
Please add CHECK-LABEL lines for the `define` lines for each of the method definitions containing these ivar accesses.  You can look at test/CodeGenObjC/direct-methods.m for the pattern I'm looking for here.